console/src/lib/tasks/NumberInput.svelte

```diff
@@ -24,7 +26,9 @@
 		fieldValidators.push(required);
 	}
 
-	let fieldValue = param.default;
+	const valueFromUrl = $page.url.searchParams.get(param.name);
+	let fieldValue = valueFromUrl ? Number(valueFromUrl) : undefined || param.default;
```

Comment:
This should have the same result (but test):

```suggestion
	let fieldValue = valueFromUrl ? Number(valueFromUrl) : param.default;
```